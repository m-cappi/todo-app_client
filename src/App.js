import React from "react";
import { BrowserRouter } from "react-router-dom";
import { Provider } from "react-redux";
import store from "./redux/store";
import IndexRouter from "./routes/IndexRouter";
import "./scss/_base.scss";

const App = () => {
  return (
    <BrowserRouter>
      <Provider store={store}>
        <IndexRouter />
      </Provider>
    </BrowserRouter>
  );
};

export default App;
