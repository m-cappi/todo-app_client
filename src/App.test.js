import React from "react";
// import { render, screen } from "@testing-library/react";
import { create } from "react-test-renderer";
import App from "./App";

test("App snapshot test", () => {
  const tree = create(<App />);
  expect(tree.toJSON()).toMatchSnapshot();
});
