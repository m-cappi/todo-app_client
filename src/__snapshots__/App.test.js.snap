// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
Array [
  <header>
    <h1>
      Mi Lista de Tareas
    </h1>
  </header>,
  <footer>
    <p>
      by 
      <a
        href="https://github.com/nepha199"
      >
        Martin Cappi Dahl
      </a>
    </p>
  </footer>,
]
`;
